src/templates/blog-post.js

```diff
@@ -16,17 +16,9 @@ function BlogPostTemplate({ data, pageContext: { previous, next }, location }) {
   return (
     <Layout location={location} title={siteTitle}>
       <SEO title={post.frontmatter.title} description={post.excerpt} />
-      <h1
-        style={{
-          marginTop: rhythm(1),
-          marginBottom: 0,
-        }}
-      >
-        {post.frontmatter.title}
-      </h1>
+      <h1>{post.frontmatter.title}</h1>
```

Comment:
Same here. I also wanted the date line below to stay closer to the title so it didn’t stand out as its own thing, since it’s less important.

Reply:
Added the bottom margin back.